src/lib/forms-schema/elements/BsbElement.ts

```diff
@@ -16,5 +17,7 @@ export default Joi.object({
   label,
   required,
   readOnly,
+  placeholderValue,
```

Comment:
I dont think we actually want this. Ill show you the mask thing ive been looking into.